cleanlab/count.py

```diff
@@ -115,10 +117,10 @@ class 0, 1, ..., K-1. `pred_probs` should have been computed using 3 (or
         # Original non-calibrated counts of confidently correctly and incorrectly labeled examples.
         confident_joint = compute_confident_joint(
             labels=labels, pred_probs=pred_probs, calibrate=False
-        )
+        )  # type: ignore
 
     if estimation_method is "off_diagonal":
-        num_issues = np.sum(confident_joint) - np.trace(confident_joint)
+        num_issues = np.sum(confident_joint) - np.trace(confident_joint)  # type: ignore
```

Comment:
why do we have to ignore this type? Isn't it clearly a scalar?

Reply:
consider asking for help if you're unsure how to get mypy to allow this to pass type-checker. If you've already asked Elias and Anish about this and this was the recommended strategy, then 👍 from me too!

Reply:
Fixed: https://github.com/aditya1503/cleanlab/blob/b5e45d622f48959292bc9ec1cc5842acfcee53e9/cleanlab/count.py#L120